ols/app/metrics/token_counter.py

```diff
@@ -101,6 +101,16 @@ def tokens_count(self, text: str) -> int:
         """Compute tokens count for given input text."""
         return len(self.token_handler.text_to_tokens(text))
 
+    def __str__(self) -> str:
+        """Textual representation of GenericTokenCounter instance."""
+        return (
+            f"{self.__class__.__name__}: "
+            + f"input_tokens: {self.input_tokens} "
```

Comment:
Out of curiosity, why do we use f strings here when we've been removing f strings from almost everywhere. Makes it more readable?

Reply:
ah yes, we removed them from logging statements. Why? Because if logging is disabled or set to value like FATAL, all those "lower level" logging like log.DEBUG will still be evaluated. Not this one, because it is evaluated just when somebody actually PRINTS the value. I mean - this statement is in method and it is not evaluated when the method is not called.

Reply:
ok, so for example in test cases, it's ok to continue using f strings since it's more readable no?
From what I remember reading, it's slightly slower tho right? Since it's further from the C integration?